codegen.yml

```
@@ -0,0 +1,10 @@
+# Would be better to get the schema from the Studio registry once it can be a public variant.
+schema: https://graphql.api.apollographql.com/api/graphql
+generates:
+  ./src/language-server/graphqlTypes.ts:
+    documents:
+      - src/language-server/providers/schema/engine.ts
+      - src/language-server/engine/operations/schemaTagsAndFieldStats.ts
```

Can we replace this with something more generic? Eg
```suggestion
      - src/**/*.ts
      - "!src/**/__tests__**/*.ts"
```